subt/ros/proxy/ros_proxy_node.cc

```diff
@@ -305,10 +305,6 @@ Controller::Controller(const std::string &_name)
   ros::service::waitForService("/subt/start", -1);
   ros::service::waitForService("/subt/pose_from_artifact_origin", -1);
 
-  // Waiting from some message related to robot so that we know the robot is already in the simulation
-  ROS_INFO_STREAM("Waiting for " << this->name << "/front/depth");
-  ros::topic::waitForMessage<sensor_msgs::Image>(this->name + "/front/depth", this->n);
-
   ROS_INFO_STREAM("Sleeping 1 simulated second to let simulation start up");
   ros::Duration(1).sleep();
   ROS_INFO_STREAM("Simulation hopefully up and running");
```

Comment:
This is also no longer needed, since we now wait for the simulation to get up and running within the startup script.

https://github.com/robotika/osgar/blob/633a5f4323fe4e2b9eaae78d964a3faf327b1fbe/subt/docker/robotika/run_solution.bash#L13

and https://github.com/robotika/osgar/blob/master/subt/ros/proxy/wait.py